solidity/contracts/SisypheanExchange.sol

```diff
@@ -297,6 +302,25 @@ contract SisypheanExchange is ForkedERC1155 {
 
 		childUniverse.ethBalance += msg.value;
 		childUniverse.ethBalanceDelta = 0;
+		childUniverse.auctionFinished = true;
+		childUniverse.ethForCash = 1 ether;
 		universes[childUniverseId] = childUniverse;
 	}
+
+	function triggerAuctionFinished(uint192 forkingUniverseId) external {
+		Universe memory forkingUniverse = universes[forkingUniverseId];
+		uint256 auctionEndTime = forkingUniverse.forkTime + REP_MIGRATION_WINDOW + AUCTION_DURATION;
+		require(block.timestamp > auctionEndTime, "Universe not finished with Auction");
+
+		// Some cash may have migrated. Since migration sends an equal balance to each universe its sufficient to just get the supply on the invalid child to account for this
+		uint256 totalCashSupply = totalSupply(forkingUniverseId) + totalSupply((forkingUniverseId << 2) + 1);
+
+		for (uint8 i = 1; i < Constants.NUM_OUTCOMES + 1; i++) {
```

Comment:
Is Solidity smart enough to inline constants from imported contracts?

Reply:
I would guess so at this point